instrumentation/log/http_output.go

```diff
@@ -0,0 +1,95 @@
+package log
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/pkg/errors"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+)
+
+type row struct {
+	level     string
+	timestamp time.Time
+	message   string
+	fields    []*Field
+}
+
+type httpOutput struct {
```

Comment:
is this really an `httpOutput?` I think it's a `bufferingOutput` with an `httpWriter`. Or maybe even `logzioHttpWriter` - unless you think it's safe to assume that other log services will use the same http API

Reply:
Done.